### PR TITLE
Fix cURL command to get a sandbox token in try_the_api.html

### PR DESCRIPTION
**Pull request by greg-pf**
Remove whitespace from sandbox token cURL command. Copy/Paste of this command failed due to the newline character

## 🎫 Ticket

~https://jira.cms.gov/browse/BCDA-xxx~

## 🛠 Changes

Remove whitespace from sandbox token cURL command. Copy/Paste this command was broken due to the newline

## ℹ️ Context for reviewers

@laurenkrugen-navapbc caught this broken command when walking through the documentation. The same command is used in [the build instructions](https://github.com/CMSgov/bcda-static-site/blob/deac7639b912dc016e30171a34d990a637be1ff7/_includes/build/access_token.html#L46-L48)

## ✅ Acceptance Validation

@laurenkrugen-navapbc has validated this cURL command during connectathon prep work

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

**Review by kyeah — APPROVED** [review]
oh! thanks :)

File: _includes/guide/try_the_api.html [hunk 1]
```diff
@@ -74,7 +74,6 @@ <h3>
 	Sample cURL Command to Submit Credentials for an Access Token
 </h3>
 <pre><code>curl -d '' -X POST 'https://sandbox.bcda.cms.gov/auth/token' \
-
 	--user 2462c96b-6427-4efb-aed7-118e20c2e997:8e87f0ebc50d10f1bc9734329a9900179b84ccd39e4d0920b905cc359cf6e94a6e760bbe3a0890c7 \
```
Comment (laurenkrugen-navapbc) [inline _includes/guide/try_the_api.html:77]: @greg-pf it's hard to tell in the github UI; is there a tab before `--user` and before the next `-H`? if so, could we also remove those? 😺

Comment (greg-pf) [reply _includes/guide/try_the_api.html:77]: would the tabs also need to be removed in [the build instructions](https://github.com/CMSgov/bcda-static-site/blob/deac7639b912dc016e30171a34d990a637be1ff7/_includes/build/access_token.html#L46-L48)?

Comment (greg-pf) [reply _includes/guide/try_the_api.html:77]: @laurenkrugen-navapbc I confirmed that the command as formatted in the build instructions can be executed with the tabs before `--user` and `-H`. The updated command in this PR is an exact match to that one

**Pull request merged**